### PR TITLE
instructions and dockerfile for building dynamodb static binaries

### PR DESCRIPTION
**Pull request by jeremyd**

### CODE REVIEW

File: build.assets/Dockerfile.dynamodb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	&& apt-get install -q -y locales) ; 
+
+# Set locale to en_US.UTF-8
+RUN (locale-gen \
+	&& locale-gen en_US.UTF-8 \
+	&& dpkg-reconfigure locales)
+
+RUN apt-get -y update && apt-get -y upgrade 
+RUN apt-get install -q -y \
+         libsqlite3-0 \
```
Comment (klizhentas) [inline build.assets/Dockerfile.dynamodb]: what do you need sqlite3 for ?

Comment (klizhentas) [inline build.assets/Dockerfile.dynamodb]: ah, disregard this is a builder dockerfile

**Review by klizhentas — COMMENTED** [review]
this looks good to me. @kontsevoy any feedback?

**Comment by jeremyd** [comment]
Hey sorry, this isn't quite ready.  I'm pushing one last commit that has
the stuff for the build assets zipfile (testing now).  Thanks!

**Comment by jeremyd** [comment]
All good now.

**Review by kontsevoy — APPROVED** [review]
Looks good, thanks! I will probably add my own comments to tie everything together, but at least there's now enough material.

**Comment by klizhentas** [comment]
ok to test

**Pull request merged**